components/script/dom/document.rs

```diff
@@ -2092,7 +2093,7 @@ impl Document {
             },
             LoadType::PageSource(_) => {
                 if self.has_browsing_context && self.is_fully_active() {
-                    // Note: if the document is not fully active, the layout thread will have exited already.
+                    // Note: if the document is not fully active, layout will have exited already.
```

Comment:
Is this comment relevant? That is, what does it mean for layout to have exited in this new architecture?

Reply:
In this case it means that the Constellation has sent an `ExitNow` message to layout. I'm not sure if it's still a relevant situation, but I think that this change is a preliminary step toward removing the `ExitNow` message and perhaps simplifying this bit of code. The thing is that this requires a bit of experimentation and testing to make sure we don't unintentionally break anything, so I think it makes sense to do this as a followup (along with many other cleanups this unlocks).